docs/blog/hacktoberfest.md

```diff
@@ -0,0 +1,17 @@
+---
+author: 'Ce Gao'
+avatar: 'https://www.github.com/gaocegege.png'
+github: 'https://github.com/gaocegege'
+twitter: 'https://twitter.com/gaocegege'
+introduction: 'Do cool things that matter | Co-founder @tensorchord | Co-chair @kubeflow | ex-@Tencent, @bytedance, @caicloud'
+---
+
+# envd is participating Hacktoberfest!
+
+[Hacktoberfest](https://hacktoberfest.com/) is an open source festival celebrated during October every year, encouraging people worldwide to actively participate and contribute to participating open source projects hosted across GitHub and GitLab.
+
+[envd](https://github.com/tensorchord/envd) is participating Hacktoberfest this year! There are some [first contributor friendly issues](https://github.com/tensorchord/envd/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue+%E2%9D%A4%EF%B8%8F%22).
```

Comment:
Does hacktoberfest have a dedicated label so it's more easily searchable? 

We can add `hacktoberfest` to the list of topics of the repo so that the project would appear here: https://github.com/topics/hacktoberfest

Reply:
> We can add hacktoberfest to the list of topics of the repo so that the project would appear here: https://github.com/topics/hacktoberfest

Thanks for the advice. We already added it!